lib/src/model/settings/board_preferences.dart

```diff
@@ -110,7 +119,9 @@ class BoardPrefs with _$BoardPrefs implements Serializable {
     required bool boardHighlights,
     required bool coordinates,
     required bool pieceAnimation,
-    required bool showMaterialDifference,
+    // required bool showMaterialDifference,
+    required MaterialDifference materialDifference,
```

Comment:
Calling this `MaterialDifferenceFormat` (like in the commented line below) would be more readable I think. Currently, there's both `MaterialDiff` and `MaterialDifference` which is confusing